src/main/java/org/wildfly/security/_private/ElytronMessages.java

```diff
@@ -1721,39 +1721,6 @@
 
     /* credential.store. package */
 
-    @Message(id = 9501, value = "Credential store '%s' has to be initialized before the first usage")
```

Comment:
Nitpick: Since these messages have not been part of a release yet, it could be good to compact them